documentation/modules/managing/con-broker-config-properties.adoc

```diff
@@ -250,81 +273,67 @@ log.segment.delete.delay.ms=60000
 # ...
 ----
 
-== Removing log data with cleanup policies
+Or configure `file.delete.delay.ms` at the topic level.
 
-The method of removing older log data is determined by the _log cleaner_ configuration.
-
-The log cleaner is enabled for the broker by default:
+You set the frequency at which the log is checked for cleanup in milliseconds:
 
 [source,env]
 ----
 # ...
-log.cleaner.enable=true
+log.retention.check.interval.ms=300000
 # ...
 ----
 
-The log cleaner needs to be enabled if you are using log compaction cleanup policy.
-You can set the cleanup policy at the topic or broker level.
-Broker-level configuration is the default for topics that do not have policy set.
+Adjust the log retention check interval in relation to the log retention settings. 
+Smaller retention sizes might require more frequent checks. 
+The frequency of cleanup should be often enough to manage the disk space but not so often it affects performance on a topic.
```

Comment:
I think `performance on a topic` could be changed to `performance on the broker` as compaction is expensive in terms of CPU, memory and disk IO.